templates/aws/cluster_nodes/manifest.yaml

```diff
@@ -42,7 +42,7 @@ variables:
     optional: false
     description: "The vpc of where the target group is located"
   aws_volume_size:
-    type: string
+    type: integer
```

Comment:
why are we changing this to an integer? Does the terraform accept both types?

Reply:
https://github.com/rancherlabs/corral-packages/issues/15

Reply:
Ok but "50G" is not a valid value

Reply:
Let's change it sorry, I needed to catch up